Web/Redirection_Based_Privilege_Escalation.md

```diff
@@ -0,0 +1,33 @@
+/*
+Title: Redirection Based Privilege Escalation
```

Comment:
That's because I coined it.

I was thinking I should write up a blog post about it and put that as the reference to the vuln here.  Sound good?

Basically, it describes a process by which an attacker can upgrade any simple xss into full control over the application.  Something that I've had to explain to clients before.